app/uk/gov/hmrc/helptosavefrontend/views/govuk_wrapper.scala.html

```diff
@@ -56,20 +57,33 @@
     <![endif]-->
 }
 
-
-
 @pageScripts = {
   @scriptElem
   <script src='@routes.Assets.versioned("javascripts/show-hide-content.js")'></script>
   <script src='@routes.Assets.versioned("javascripts/do-not-track.js")'></script>
+  <script>
+      window.jsConfig = window.jsConfig || {};
+      window.jsConfig.timeoutEnabled = @if(isSignedIn){ true } else {false};
```

Comment:
can this not be simplified to `window.jsConfig.timeoutEnabled = @{isSignedIn};`

Reply:
in fact why can we not use `enableSignOutLink` which looks like it already exists to test signed in status?

Reply:
@nubz yes right fixed, please review it.